linetools/isgm/abssystem.py

```diff
@@ -223,6 +224,42 @@ def chk_component(self, component):
         """
         return True
 
+    def get_absline(self, inp):
+        """ Returns an AbsLine from the AbsSystem
+
+        Parameters
+        ----------
+        inp : str or Quantity
+          str -- Name of the transition, e.g. 'CII 1334'
+          Quantity -- Rest wavelength of the transition, e.g. 1334.53*u.AA
+            to 0.01 precision
+
+        Returns
+        -------
+        absline -- AbsLine object or list of Abslines
+          More than one will be returned if this line exists in
+          multiple components.  The returned quantity will then
+          be a list instead of a single object
+        """
+        # Generate the lines
+        abslines = self.list_of_abslines()
+        if isinstance(inp,basestring):
+            names = np.array([absline.trans for absline in abslines])
```

Comment:
Should we change the attribute `trans` in AbsLines for `name` instead? (Not sure though)